### ISSUE TITLE
rpm spec missing or docs incorrect

### ISSUE THREAD

**Issue by bdeetz** [irods-contrib/metalnx-web#65]
https://github.com/irods-contrib/metalnx-web/wiki/Package-Metalnx#create-rpm-and-deb-packages claims that `packaging/rpm/emc-metalnx-webapp.spec` exists. It does not. Should the wiki be updated to only include docker build instructions?

**Comment by michael-conway** [comment]
Yes, there's still cleanup to do, we'll make sure that's in for the release
of 2.0.0

**Comment by michael-conway** [comment]
@mcoonen can we chat about your updates and what you'd like to see merged in to get to a master? With the new year and some projects complete we're jumping in for our own release. Ok @trel?

**Comment by trel** [comment]
Yeps

**Comment by mcoonen** [comment]
Hi @michael-conway and @trel,

Our _MaastrichtUniversity/metalnx-web_ fork only concerns commits for easier deployment via Docker compose. That work is rather specific to our Maastricht situation and is not necessarily useful for others. 

With your message you might refer to the commits "pushed by me" in the thread above. I just merged my Maastricht fork with the changes from irods-contrib/metalnx-web. Apparently those commits (which in fact are yours) end up in this thread with my name?!
I don't know how that happened... sorry about that

**TL;DR:**  I didn't push any updates regarding issue 65, so please ignore those. Those are updates made by you.

**Comment by mcoonen** [comment]
On the other hand, I would like the breadcrumb feature, but I see that you already opened issue 77 for that! 👍

**Comment by michael-conway** [comment]
I'm going to remove the doc for now and we can revisit later. Docker seems to be the consensus winner on deployment styles.
